helm/templates/view-creator-job-config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.viewCreatorJobConfig.name }}
+  labels:
+    release: {{ .Release.Name }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
On top of this I had to add a `"helm.sh/hook-weight": "-5"` to get it loaded before the jobs. Were you able to get it to work without the weights?

Reply:
I can get it to work w/o weights. As in https://github.com/helm/helm/blob/master/pkg/releaseutil/kind_sorter.go#L31, Helm sorts resources by Kind, Job is after ConfigMap.